examples/plot_bootstrap_ci.py

```diff
@@ -0,0 +1,97 @@
+"""
+=======================================================
+Plot bootstrapped confidence interval for condition ERP
+=======================================================
+
+"""
+
+# Authors: Jose C. Garcia Alanis <alanis.jcg@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from mne.viz import plot_compare_evokeds
+from mne.decoding import Vectorizer
+from mne.datasets import limo
+
+###############################################################################
+# list with subjects ids that should be imported
+subjects = [2]
+# create a dictionary containing participants data
+limo_epochs = {str(subj): limo.load_data(subject=subj) for subj in subjects}
+
+# get key from subjects dict for easy slicing
+# subjects = list(limo_epochs.keys())
+
+# interpolate missing channels
+for subject in limo_epochs.values():
+    subject.interpolate_bads(reset_bads=True)
+
+###############################################################################
+# data to be analysed
+face_a = limo_epochs['2']['Face/A'].get_data()
+
+# number of trials per condition
+n_epochs = face_a.shape[0]
+
+# number of channels and number of time points in each epoch
+# we'll this this information later to bring the results of the
+# the linear regression algorithm into an eeg-like format
+# (i.e., channels x times points)
+n_channels = len(limo_epochs['2'].info['ch_names'])
+n_times = len(limo_epochs['2'].times)
+
+# vectorize (channel) data for linear regression
+Y = Vectorizer().fit_transform(face_a)
+
+###############################################################################
+# run bootstrap for centrality estimator
+
+# set random state for replication
+random_state = 42
+random = np.random.RandomState(random_state)
+
+# number of random samples
+boot = 2000
+
+# initialize bootstrap
+resampled_data = []
+for i in range(boot):
+    # pick random samples with replacement from data
+    resamples = random.choice(range(n_epochs), n_epochs, replace=True)
+    # compute centrality estimator on re-sampled data
+    resampled_data.append(Y[resamples, :].mean(axis=0))
```

Comment:
I think it would be more pedagogical to show how to bootstrap the fit, even if this may give identical results in this case.

Reply:
Ok, so then perhaps re-open #6 and close this one?

Reply:
I think we/me got confused with all these prs. Well or copy over the few lines meded.